frontend/src/views/Lobby.tsx

```diff
@@ -58,9 +58,11 @@ function LobbyPage() {
     setDifficulty(room.difficulty);
   };
 
-  const deleteUser = (user: User) => {
-    // Make rest call to delete user from room
-    console.log(user);
+  const deleteUser = (userId: string) => {
+    removeUser(currentRoomId, userId)
+      .catch((err) => {
+        setError(err.message);
+      });
```

Comment:
nit: Add the usual `setLoading` logic here so the host knows the request is being processed